docker/config/my.env.dist

```diff
@@ -16,4 +16,4 @@ OAUTH2_CLIENT_SECRET=
 
 # API token from crash-stats.mozilla.com that has access to download
 # raw dumps
-API_TOKEN=
+SOCORRO_API_TOKEN=
```

Comment:
I renamed this to make it clearer as to what it was an API token for. I'm game for other naming ideas.